### DISCUSSION TITLE
are these bugs or features?

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22195] [Q&A] [answered]
if they are infact bugs, should I add them to the issues list? all issues are on current version, ill add it if I actually create issues.

First, when using a black or really dark cursor(the one in vid is RGB 8,8,8(yes barely anybody would play with that but thats not the point)), you can't see it very well after pausing and resuming gameplay, stable handles this very differently.

https://user-images.githubusercontent.com/87947227/212444899-c379baa2-f405-4525-95f9-04dfd34ffd06.mp4

https://user-images.githubusercontent.com/87947227/212445032-b1a4611c-cfb8-4c5d-9f1b-2142331c58bb.mp4

.
.
.
.
.
.
Next when skinning the songeselect there is a deadzone in some areas, where you effectively cannot place elements by conventional means. I colored It white to clarify what I mean. Its generally pretty annoying that nothing ever ends up where you place it, but well, I found a workaround.

https://user-images.githubusercontent.com/87947227/212445125-9f70322a-0207-4cab-aca8-7ff391c699cc.mp4

as you can see rescaling the element doesnt change the "Anchor" so you can do funny stuff like this

https://user-images.githubusercontent.com/87947227/212445147-35814372-790b-4e33-b596-4a131b7c81bd.mp4

.
.
.
.
.
.
Last the multiplayer spectate suddenly went all transparent even though my dim is at 100%, and when maximizing certain players ist looks like this. This happened after I left a lobby and then joined back in and started spectating.

https://user-images.githubusercontent.com/87947227/212445352-92924b75-9285-4044-baa5-08a159139b62.mp4

**Comment by peppy** [comment, marked as answer]
Too many things in one post. They are probably unintended but skins and what not. And maybe bugs for your other issues. If you want us to look into it further please open one discussion for each issue.

**Reply by Inconsist** [reply]
can I delete this? Or do you have to do that?
